packages/smooth_app/lib/helpers/string_extension.dart

```diff
@@ -0,0 +1,5 @@
+extension StringExtension on String {
+  String capitalize() => isEmpty
+      ? this
+      : this[0].toUpperCase() + (length == 1 ? '' : substring(1));
```

Comment:
You can simplify, but replacing `(length == 1 ? '' : substring(1)` with just `substring(1)`

Reply:
@g123k I'll try that, but I would be surprised if `substring` accepted a starting index beyond the string.

Reply:
@g123k Fair enough, it does work! I still don't see the logic behind it though...